python/two-fer/two_fer.py

```diff
@@ -1,2 +1,6 @@
-def two_fer(name):
-    pass
+def two_fer(name = None):
+    if name == None:
+        return 'One for you, one for me.'
+    else:
+        return  'One for ' + name + ', one for me.' 
```

Comment:
This works. Can we have it a bit clearer using the `str.format()`? Have a look at the docs